applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py

```diff
@@ -84,7 +83,6 @@ def GetDefaultParameters(cls):
                 "input_type": "mdpa",
                 "input_filename": "unknown_name"
             },
-            "computing_model_part_name" : "thermal_computing_domain",
```

Comment:
May be we can issue a deprecation warning here?

Reply:
Considering that this was silently set by the solver defaults (no GUI was giving the option to customize this) I don't think it's worth it in this case.